flexmeasures/data/migrations/versions/a60cc43aef5e_delete_children_assets_on_cascade_when_.py

```diff
@@ -0,0 +1,46 @@
+"""Delete children assets on cascade when deleting an asset.
+
+Revision ID: a60cc43aef5e
+Revises: ac5e340cccea
+Create Date: 2023-10-11 14:04:19.447773
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "a60cc43aef5e"
+down_revision = "ac5e340cccea"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_constraint(
+        "generic_asset_parent_asset_id_generic_asset_fkey",
+        "generic_asset",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        None,
+        "generic_asset",
+        "generic_asset",
+        ["parent_asset_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
```

Comment:
The class now doesn't set the same `ondelete` value, it seems?

```
parent_asset_id = db.Column(
    db.Integer, db.ForeignKey("generic_asset.id"), nullable=True
)
```

Reply:
True, I forgot to add it back when I dropped it for a test.